docs/cnv-configuration.md

```diff
@@ -0,0 +1,153 @@
+`requests.cpu`, `limits.cpu`, `requests.memory`, `limits.memory`, and more. 
+
+Openshift-Virtualization allow the deployment and management of virtual machines alongside
+containerized applications.
+
+Administrators of multi-tenant clusters running Virtual Machines with OpenShift Virtualization 
+might expect tenants to be billed only for the resources explicitly requested in the VM object. 
+However, this presents specific challenges with resource quotas:
+
+1. VM Live Migration: VM instances run inside pods. Occasionally, the pod associated with a VM needs to
+   be evicted from one node and rescheduled onto another. VMs have a large state that requires careful
+   handling during migration. To manage this, a new pod is created to transfer the VM state before
+   terminating the old one, which differs from how standard pods are moved. Running both pods simultaneously
+   temporarily doubles the amount of resources the tenant is accounted for and can impact resource quotas,
+   potentially leading to migration issues and upgrade failures.
+2. Each Virtual Machine operates within its own container. This means that the virtualization overhead, 
+   runs within each VM pod instead of centrally on the host, as in traditional virtualization. 
+   This overhead is factored into quota calculations. In traditional virtualization platforms,
+   service providers typically base VM quotas solely on RAM and CPU threads, since users don't directly request
+   resources for this overhead or expect its effects. Moreover, the resource requirements for this overhead can
+   change between versions without users being notified.
+
+As a result, deploying Virtual Machines in a multi-tenant cluster while restricting hardware consumption
+from different entities is highly challenging. It requires significant resource allocation to
+handle live migrations and manage VM infrastructure overhead. This also couples users to the
+VM implementation, making them dependent on infrastructure overhead that can change unexpectedly.
+
+### Solution: AAQ Operator
+By default, AAQ Operator allocates resources for VM-associated pods, appending a `/vmi` suffix to
+`requests/limits.cpu` and `requests/limits.memory` compute resources, derived from the VM's RAM size and CPU threads.
+
+For both limits and requests with the `/vmi` suffix, the same values will apply because, unlike pods, virtual
+machines are already limited to the virtual memory and virtual CPU that were allocated to them.
+
+### Application Aware Resource Quota
+The ApplicationAwareResourceQuota API is compatible with the native ResourceQuota, meaning it shares the same specification and status definitions.
+
+Example manifest:
+
+```yaml
+apiVersion: aaq.kubevirt.io/v1alpha1
+kind: ApplicationAwareResourceQuota
+metadata:
+  name: example-resource-quota
+spec:
+  hard:
+    requests.memory: 1Gi
+    limits.memory: 1Gi
+    requests.cpu/vmi: "1"
+    requests.memory/vmi: 1Gi
```

Comment:
In most situations, wouldn't we set `requests.memory` above `requests.memory/vmi`? i.e. wouldn't it be more realistic?

Reply:
Please remember that `requests.memory/vmi` and `requests.memory` are counted separately. `requests.memory` is for pods not associated with virtual machines, while `requests.memory/vmi` is only for pods associated with virtual machines, where we count the VM's RAM.

So, when we set `requests.memory: 1Gi` and `requests.memory/vmi: 1Gi`, it means that pods not associated with virtual machines can use up to 1Gi, and relevant VMs can use up to 1Gi of RAM.

I hope this answers your question.